src/containers/Society.jsx

```diff
@@ -148,6 +155,7 @@ class Society extends Component {
 
 const mapStateToProps = state => ({
   societyInfo: state.societyInfo,
+  requesting: state.societyInfo.requesting,
```

Comment:
Since you are using `societyInfo. requesting `, you should remove this as you are not using it.